src/cli/restore/sharepoint.go

```diff
@@ -64,6 +66,16 @@ func addSharePointCommands(cmd *cobra.Command) *cobra.Command {
 			utils.LibraryItemFN, nil,
 			"Restore library items by file name or ID")
 
+		fs.StringSliceVar(
+			&listPaths,
+			utils.ListFN, nil,
+			"Restore list items by SharePoint path")
```

Comment:
`by path` won't be very clear to the user.  Maybe this is better?
```suggestion
			"Restore list items by SharePoint list ID")
```